register/js/script.js

```diff
@@ -62,7 +62,7 @@ $(document).ready(function()  {
     if($("#right").hasClass("col-md-6")){
         $(this).toggleClass("col-md-6 col-md-12",1000);
     }
-    $("#left").hide();
+    $("#left").hide(1000);
```

Comment:
Not a correct fix i suppose. If that worked for you, post a video.

Reply:
I think you got confused. This will actually slow down closing speed. We want to fix speed when it shows up again after clicking exit button. Just do a change as you did below. Remove this change.